package.json

```diff
@@ -26,6 +26,7 @@
     "react-dom": "^17.0.1"
   },
   "dependencies": {
+    "@headlessui/react": "^0.3.2",
```

Comment:
`@headlessui/react` can be updated from `0.3.2` to `1.0.0`, as they release a few more components.

Reply:
Nice! let's do that